src/app/components/HorizontalRule/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { C_POSTBOX } from '../../lib/constants/styles';
+
+const HorizontalRule = styled.hr`
+  background-color: ${C_POSTBOX};
+  height: 4px;
```

Comment:
I think the Horizontal rule should scale with the font-size scaling, since otherwise it could look comically small in comparison to large text above and below.